bigflow/cli.py

```diff
@@ -514,7 +530,7 @@ def _resolve_vault_endpoint(args):
 def _resolve_property(args, property_name, ignore_value_error=False):
     try:
         cli_atr = getattr(args, property_name)
-        if cli_atr:
+        if cli_atr or cli_atr is False:
```

Comment:
Is else part now even reachable?

Reply:
Yup, when you use `--vault-endpoint-verify false`

Reply:
`getattr` can return `None`, and then it reaches the else part